src/common/styles.c

```diff
@@ -593,11 +593,13 @@ void dt_styles_apply_style_item(dt_develop_t *dev, dt_style_item_t *style_item,
               && dt_develop_blend_legacy_params(module, style_item->blendop_params, style_item->blendop_version,
                   module->blend_params, dt_develop_blend_version(), style_item->blendop_params_size) == 0)
       {
+        // legacy style
         // do nothing
       }
       else
       {
-        memcpy(module->blend_params, module->default_blendop_params, sizeof(dt_develop_blend_params_t));
+        // apply style on export
+        memcpy(module->blend_params, style_item->blendop_params, sizeof(dt_develop_blend_params_t));
```

Comment:
I don't think this is the right fix. This else part is when the blend_param from the style does not correspond to the current blend version. So we cannot use style_item blendop_params and we just set to default blend.

The case where the version matches is in the first part of the if statement above.